### PR TITLE
iJento tracking needs to work, specifically if the browser *fails* CTM

### PR DESCRIPTION
**Pull request by adambraimbridge**
iJento tracking needs to work, specifically if the browser *fails* the cuts-the-mustard test

### CODE REVIEW

File: templates/tracking.html [hunk 1]
```diff
@@ -1 +1,34 @@
...
+However, a copy remains here because iJento tracking needs to work,
+specifically if the browser *fails* the cuts-the-mustard test.
+
+... if the CTM test passes, the iJento code here is not included, as
+it's handled in `n-third-party-tools`.
+}}
+{{#if @root.flags.javascript}}
+	{{#if @root.flags.analytics}}
+	<script>
+		var cutsTheMustard = 'getComputedStyle' in window && !(window.navigator.userAgent.indexOf('IEMobile') > -1 && !CustomEvent);
```
Comment (wheresrhys) [inline templates/tracking.html]: don't think you need this line as the test is already done in the ctm.html template, and the result stored as a global variable

Comment (adambraimbridge) [inline templates/tracking.html]: I could detect the absense of the `js` class that's added in https://github.com/Financial-Times/next-js-setup/blob/master/templates/ctm.html#L8 ... but that seems odd somehow. 

I couldn't find a global CTM variable, but yes that'd be much better than having dupe CTM code here.

**Comment by adambraimbridge** [comment]
<facepalm>
image

**Comment by ironsidevsquincy** [comment]
just out of interest, what's the benefit of doing this rather than just always doing it here and removing ijento from `n-third-party-tools`

File: templates/tracking.html (hunk 1 above)
Comment (adambraimbridge) [inline templates/tracking.html]: Update: I'm a derp head

**Comment by adambraimbridge** [comment]
@ironsidevsquincy The goal of n-third-party-code is to centralise all the tracking, advertising (ie third-party) code into one component, so that future requests for more third-party scripts can be managed.

**Comment by ironsidevsquincy** [comment]
but aren't you de-centralising it by having it here as well? get the benefit of it, just seems a bit odd to duplicate in two places for the sake of centralisation

just wondering if there's a way of keeping it in one place, and have a non-ctm route that can call it

**Comment by adambraimbridge** [comment]
Completely agree. I jumped the gun by deleting this code entirely, then after deploying n-third-party-tools, mattc noted this would make iJento blind to non-CTM browsers. So I've put it back here cause it's where it was originally, and once I figure out how to execute to non-CTM from within the n-third-party-code component, I'll delete it from here.

**Comment by ironsidevsquincy** [comment]
got it, thanks

**Comment by wheresrhys** [comment]
there's nothing to stop next-js-setup's script loader consuming a template provided by n-third-party

**Comment by adambraimbridge** [comment]
@wheresrhys Thanks. Sounds like I won't have too much hassle sorting this out next week.

**Pull request merged**